### PR TITLE
jyg-uId2-1709 update example app to use v2

### PR DESCRIPTION
**Pull request by jingyi-gao-ttd**

### CODE REVIEW

File: .github/workflows/docker-image.yml
Comment (alex-yau-ttd) [inline .github/workflows/docker-image.yml:1]: should we rename this file too?

File: examples/google-esp-integration/with_sdk_v3/README.md [hunk 1]
```diff
@@ -15,17 +15,19 @@ docker build . -t uid2-esp-standard
 docker run -it --rm -p 3000:3000 \
     -e UID2_BASE_URL="https://operator-integ.uidapi.com" \
     -e UID2_API_KEY="<your-integ-API-key>" \
+    -e UID2_CLIENT_SECRET="{CLIENT_SECRET}" \
     -e AD_TAG_URL="<your-IMA-ad-tag-url>" \
```
Comment (alex-yau-ttd) [inline examples/google-esp-integration/with_sdk_v3/README.md:19]: we don't need SESSION_KEY here?

Comment (jingyi-gao-ttd) [reply examples/google-esp-integration/with_sdk_v3/README.md:19]: No, only server-only needs SESSION_KEY

File: .github/workflows/docker-image.yml
Comment (jingyi-gao-ttd) [reply .github/workflows/docker-image.yml:1]: done

File: examples/google-esp-integration/server_only/README.md [hunk 2]
```diff
@@ -14,20 +14,22 @@ The easiest way to try the example is to use the following Docker Build command:
...
-| `UID2_BASE_URL` | string    | The base URL of the UID2 service. For example:</br>Testing environment: `https://integ.uidapi.com`<br/>Production environment: `https://prod.uidapi.com` |
-| `UID2_API_KEY`  | string    | Your UID2 authentication key for the UID2 service specified in `UID2_BASE_URL`.                                                                          |
-| `SESSION_KEY`   | string    | The key to the encryption session data stored in the application's session cookie.                                                                       |
-| `AD_TAG_URL`    | string    | The ad tag URL to test ad requests.                                                                                                                      |
+| Parameter            | Data Type | Description                                                                                                                                              |
+| :------------------- | :-------- | :------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
```
Comment (alex-yau-ttd) [inline examples/google-esp-integration/server_only/README.md:27]: I don't think it actually affects how it renders, but I don't think we need the extra bit at the end
```
 --- |
```

File: examples/google-esp-integration/server_only/README.md [hunk 3]
```diff
@@ -14,20 +14,22 @@ The easiest way to try the example is to use the following Docker Build command:
...
+| `UID2_BASE_URL`      | string    | The base URL of the UID2 service. For example:</br>Testing environment: `https://integ.uidapi.com`<br/>Production environment: `https://prod.uidapi.com` |
+| `UID2_API_KEY`       | string    | Your UID2 authentication key for the UID2 service specified in `UID2_BASE_URL`.                                                                          |
+| `UID2_CLIENT_SECRET` | string    | Your UID2 client secret for the UID2 service specified in `UID2_BASE_URL`.                                                                               |     |
```
Comment (alex-yau-ttd) [inline examples/google-esp-integration/server_only/README.md:30]: nit: extra
```
     |
```

File: examples/google-esp-integration/server_only/README.md (hunk 2 above)
Comment (jingyi-gao-ttd) [reply examples/google-esp-integration/server_only/README.md]: Good catch! Fixed

File: examples/google-esp-integration/server_only/README.md [hunk 4]
```diff
@@ -14,20 +14,22 @@ The easiest way to try the example is to use the following Docker Build command:
 docker build . -t uid2-esp-server
 docker run -it --rm -p 3000:3000 \
     -e UID2_BASE_URL="https://operator-integ.uidapi.com" \
-    -e UID2_API_KEY="<your-integ-API-key>" \
-    -e SESSION_KEY="my-session-key" \
-    -e AD_TAG_URL="<your-IMA-ad-tag-url>" \
+    -e UID2_API_KEY="{INTEG_API_KEY}" \
+    -e UID2_CLIENT_SECRET="{CLIENT_KEY}" \
+    -e SESSION_KEY="my-uid2-session" \
+    -e AD_TAG_URL="" \
```
Comment (alex-yau-ttd) [inline examples/google-esp-integration/server_only/README.md:20]: should this be empty string or `<your-IMA-ad-tag-url>` like `with_sdk_v3` is?

File: examples/google-esp-integration/server_only/README.md [hunk 5]
```diff
@@ -15,19 +15,21 @@ docker build . -t uid2-esp-server
 docker run -it --rm -p 3000:3000 \
     -e UID2_BASE_URL="https://operator-integ.uidapi.com" \
     -e UID2_API_KEY="<your-integ-API-key>" \
-    -e SESSION_KEY="my-session-key" \
+    -e UID2_CLIENT_SECRET="<your-integ-client-secret>" \
+    -e SESSION_KEY="my-uid2-session" \
```
Comment (alex-yau-ttd) [inline examples/google-esp-integration/server_only/README.md:19]: sorry, one more nit. perhaps this should read
```
-e SESSION_KEY="<your-uid2-session>" \
```
for consistency?

**Review by alex-yau-ttd — APPROVED** [review]

**Pull request merged**